src/components/EmptyState/EmptyState.stories.mdx

```diff
@@ -54,8 +54,8 @@ export const doNothing = () => {};
             name="containers"
             style={{
               opacity: 0.25,
-              height: "2.5rem",
-              width: "2.5rem",
+              height: "10rem",
+              width: "10rem",
```

Comment:
Icons shouldn't be used this way. They were not designed to be scaled that much up (and weird things may start happening).

This one for example doesn't scale on small screens due to missing max-width that we have on `img`:
<img width="407" alt="image" src="https://github.com/canonical/react-components/assets/83575/09ecd607-ce18-41ec-97fd-59ff6d1f7c22">

Ideally, this should only be used with images. Would be good not to document an example with a bad pattern like this.

Maybe let's use this one from assets: https://assets.ubuntu.com/v1/c17e0d92-container.svg
<img width="50" height="50" src="https://assets.ubuntu.com/v1/c17e0d92-container.svg" />